<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HalamanAdmin</class>
 <widget class="QWidget" name="HalamanAdmin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>801</width>
     <height>601</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget #widget {background-image: url(:/gambar/desainatmin.png);}
</string>
   </property>
   <widget class="QTableWidget" name="tabelBuku">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>721</width>
      <height>201</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Judul</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Penulis</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Penerbit</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tahun Terbit</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="tambahButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Tambah</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tampilkanButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>360</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Tampilkan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="updateButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>360</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hapusButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>360</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Hapus</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="judulEdit">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>401</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="penulisEdit">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>130</y>
      <width>401</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="penerbitEdit">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>190</y>
      <width>401</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="tahunEdit">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>250</y>
      <width>401</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
